synapse/tools/storm_format.py

```diff
@@ -0,0 +1,65 @@
+# pragma: no cover
```

Comment:
Not sure we want this as an independant tool.  Seems a bit special purpose and inconvenient.  Is this something we should integrate into cmdr functionality?

Reply:
yes it is

Reply:
Moved to lib